routes/graphql.js

```diff
@@ -107,15 +99,15 @@ class SkyblockResolver {
   }
 
   async profiles({ player_name }) {
-    const uuid = await getUUIDAsync(player_name);
+    const uuid = await getUUID(player_name);
     const profiles = await redisGetAsync(`skyblock_profiles:${uuid}`);
     return profiles ? JSON.parse(profiles) : {};
   }
 
   async profile({ player_name, profile_id }) {
-    const uuid = await getUUIDAsync(player_name);
-    const profile = await buildProfileAsync(uuid, profile_id, true);
-    const players = await populatePlayersAsync(Object.keys(profile.members).map((uuid) => ({ uuid })));
+    const uuid = await getUUID(player_name);
+    const profile = await buildProfile(uuid, profile_id);
```

Comment:
This call used to pass `true` for `shouldUpdateProfileList`, but doesn't anymore. Is that intentional?

Reply:
That's because it's the default action.